gnucash/report/trep-engine.scm

```diff
@@ -1226,6 +1228,20 @@ be excluded from periodic reporting.")
                                   "number-cell"
                                   (xaccSplitGetAmount split)))))
 
+               (add-if (column-uses? 'association)
+                       (vector ""
+                               (lambda (split transaction-row?)
+                                 (let ((url (xaccTransGetAssociation
+                                             (xaccSplitGetParent split))))
+                                   (and (not (string-null? url))
+                                        (gnc:make-html-table-cell/markup
+                                         "text-cell"
+                                         (if opt-use-links?
+                                          (gnc:html-transaction-association-anchor
+                                           (xaccSplitGetParent split)
+                                           (_ "📎"))
+                                           (_ "📎"))))))))
```

Comment:
ahem.

Reply:
Not sure what you mean by this?

Reply:
There should be only one `(_ "📎"))` -- delete line 1242

Reply:
Ah sorry didn't see your `opt-use-links?`... my bad. The indenting made it confusing.

Reply:
This is why emacs indent *strictly* according to parentheses level as follows:
```scheme
               (add-if (column-uses? 'association)
                       (vector ""
                               (lambda (split transaction-row?)
                                 (let ((url (xaccTransGetAssociation
                                             (xaccSplitGetParent split))))
                                   (and (not (string-null? url))
                                        (gnc:make-html-table-cell/markup
                                         "text-cell"
                                         (if opt-use-links?
                                             (gnc:html-transaction-association-anchor
                                              (xaccSplitGetParent split)
                                              (_ "📎"))
                                             (_ "📎"))))))))
```

Reply:
There was me just about to point that out, what should the indenting be and I will change it.

Reply:
Thinking back... if `opt-use-links?` is #FALSE then we shouldn't even display the paperclip :laughing:

Reply:
OK, I will change that and push later.

Reply:
It does show that there is an association even if they can not click on it. If the user does not want it to show like that, they just do not display the column.